scripts.d/support/create_catalog_ng.sh

```diff
@@ -67,7 +67,7 @@ for name in ${!SERVER[@]}; do
         continue
     fi
 
-    ssh -o StrictHostKeyChecking=no $name.hpc.edf.fr \
+    ssh -o StrictHostKeyChecking=no $USER@$name.hpc.edf.fr \
```

Comment:
Ok, wil merge. Note that in a case like this it is better for readability to use `{ }` to properly show what is taken as variable names:

`$USER@$name.hpc.edf.fr -> ${USER}@${name}.hpc.edf.fr`